tspng/test_extraction.py

```diff
@@ -31,23 +37,44 @@ def test_extract_from_bytes():
         test_data = extract_from_bytes(buf)
         assert list(test_data.keys())==['info','licenses','images','annotations','models','categories']
 
+def test_extract_from_bytes_fails():
+    with pytest.raises(TypeError):
+        extract_from_bytes('tests/assets/example_file_1.ts.png')
+
 def test_extract_from_file():
     '''
     Tests the dictionary keys from an example TSPNG file path.
     '''
     test_data = extract_from_file('tests/assets/example_file_1.ts.png')
     assert list(test_data.keys())==['info','licenses','images','annotations','models','categories']
 
+def test_extract_from_file_fails():
+    #test path does not exist
+    with pytest.raises(Exception):
+        extract_from_file('Random/path.png')
+    #test file does not exist
```

Comment:
Comment should be "test folder instead of file"

Reply:
Made change as requested